.env.example

```
@@ -0,0 +1,26 @@
+# db
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+
+# notification
+NOTIFICATION_SERVER_PORT=
+NOTIFICATION_DB_PORT=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NOTIFICATION_DB_PORT key should go before the NOTIFICATION_SERVER_PORT key

(UnorderedKey)